test/setup.ts

```diff
@@ -1,4 +1,6 @@
+import ResizeObserverPolyfill from 'resize-observer-polyfill'
```

Comment:
You can back out this change, because the polyfill is now imported in the implementation file, so it is not necessary to special-case it for testing.